src/mceditlib/worldeditor.py

```diff
@@ -773,6 +775,14 @@ def fillBlocksIter(self, box, block, blocksToReplace=(), updateLights=True):
 
     def fillBlocks(self, box, block, blocksToReplace=(), updateLights=True):
         return exhaust(self.fillBlocksIter(box, block, blocksToReplace, updateLights))
+    
+    # --- Analyze ---   
+
+    def analyzeIter(self, selection):
+        return AnalyzeOperation(self, selection)
```

Comment:
Not really needed here. I don't think I need to put aliases to all of the different operations onto WorldEditor.

Besides, this one needs to return something and should be called directly.